### ISSUE TITLE
Unsupported platform 'None'

### ISSUE THREAD

**Issue by davehocking** [ev3dev/ev3dev-lang-python#613]
- **ev3dev version:** 4.14.80-ev3dev-2.3.1-rpi2
- **ev3dev-lang-python version:** python3-ev3dev : 1.2.0 / python3-ev3dev2 : 2.0.0~beta2 

Hi there, I was working well on an EV3 Brick and decided it was time to move to BrickPi.
I'm struggling to get my code to run, it seems to know it's not on an EV3 Brick, but I don't know if I've missed something where I need to declare the EV3Dev Hardware type...

  File "/usr/lib/python3/dist-packages/ev3dev2/sensor/__init__.py", line 59, in <module>
    raise Exception("Unsupported platform '%s'" % platform)
Exception: Unsupported platform 'None'

Any thoughts?

Thanks,
Dave

**Comment by dlech** [comment]
can you share your code? have you searched other issues about brickpi? did you edit config.txt?

**Comment by davehocking** [comment]
Hi, apologies for my poor first post - It was 1am in the morning and I wasn't thinking properly.
I had reviewed the troubleshooting steps but hadn't seen [the notes here](http://docs.ev3dev.org/projects/lego-linux-drivers/en/ev3dev-stretch/brickpi3.html) about editing the config file. 

Thanks,
Dave

**Comment by dlech** [comment]
no worries 😄 
Did editing config.txt fix your problem?

**Comment by davehocking** [comment]
Thanks, yeah, it did. I had a couple of extra niggles with the ports not being auto-sensing, but I was able to work out what was needed to get things working. It's currently needing me to echo to /sys/proc, I've seen some examples where it's done in Python but I didn't get any joy when I tried - likely to be my issue as this is my "learning python" experience (coming from PoSH). Worth creating a new thread for this?
